### PR TITLE
Updating compatibility information on the regocpp integration

### PR DESCRIPTION
**Pull request by matajoh**
### Why the changes in this PR are needed?

The documentation on the rego-cpp integration is out of date.

### What are the changes in this PR?

I updated the compatibility information on the rego-cpp integration. Please see [rego-cpp](https://github.com/microsoft/rego-cpp) for more information if needed.

### CODE REVIEW

File: docs/website/content/integrations/regocpp.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ tutorials:
 code:
 - https://github.com/microsoft/rego-cpp
 ---
-The rego-cpp project is a Rego compiler and runtime implemented in C++.  As such, it provides the ability to integrate Rego natively into a wider range of languages. We currrently support C, C++, Rust, and Python, and are largely compatible with v0.55.0 of the language.
\ No newline at end of file
+The rego-cpp project is a Rego compiler and runtime implemented in cross-platform C++.  As such, it provides the ability to integrate Rego natively into a wider range of languages on most platforms. In addition to CMake FetchContent support for C and C++, we also distribute a Rust crate and a Python wheel. We support v0.68.0 of the rego language as well as full v1.0 compatibility.
```
Comment (srenatus) [inline docs/website/content/integrations/regocpp.md:13]: 🤔 I'm assuming you're following the latest OPA release? If so, I would rephrase this, or you'd need to update the docs every month... also, 

> v0.68.0 of the rego language

sounds a bit wrong to me. The Rego language (capital R) isn't versioned as OPA is... so, I suppose you support "Rego as of OPA v0.68.0" or "Rego, as of the latest OPA release"

Comment (matajoh) [reply docs/website/content/integrations/regocpp.md:13]: Yeah, I can probably change the wording to be latest release to avoid having to update this regularly.

**Review by srenatus — APPROVED** [review]
